### ISSUE TITLE
Editor crashes if it is destroyed just after being initialized

### ISSUE THREAD

**Issue by p2kmgcl** [liferay/alloy-editor#1306]
```
const editor = AlloyEditor.editable(...);
editor.destroy();
```

**Comment by wincent** [comment]
Those `setTimeout` are coming back to haunt us, @julien. 😂

@p2kmgcl: do you have a stack trace?

**Comment by julien** [comment]
@wincent 🙀 
@p2kmgcl maybe you need to wait for some event to be fired. (`instanceReady `)?
In anycase we'll have a look thanks for reporting.

**Comment by p2kmgcl** [comment]
@wincent @julien  I tried waiting for `instanceReady` and CKEditor still logs this error message
![image](https://user-images.githubusercontent.com/800645/62682290-1f33f600-b9bc-11e9-8b30-2f8abb9e7141.png)

**Comment by julien** [comment]
@p2kmgcl Yes, I've just seen the same thing, so I'll investigate a bit more.
It looks like #375 might be related.
